libs/domains/organizations/feature/src/lib/git-provider-setting/git-provider-setting.tsx

```diff
@@ -0,0 +1,80 @@
+import { type GitAuthProvider, type GitTokenResponse } from 'qovery-typescript-axios'
+import { Controller, useFormContext } from 'react-hook-form'
+import { useParams } from 'react-router-dom'
+import { Icon, InputSelect } from '@qovery/shared/ui'
+import { upperCaseFirstLetter } from '@qovery/shared/util-js'
+import useAuthProviders from '../hooks/use-auth-providers/use-auth-providers'
+import useGitTokens from '../hooks/use-git-tokens/use-git-tokens'
+
+export interface GitProviderSettingProps {
+  disabled?: boolean
+}
+
+export const mergeProviders = (authProviders: GitAuthProvider[], gitTokens?: GitTokenResponse[]) => {
+  const currentAuthProviders = authProviders.map((provider: GitAuthProvider) => ({
```

Comment:
```suggestion
  const currentAuthProviders = authProviders.map((provider) => ({
```
Type should not be needed here

Reply:
https://github.com/Qovery/console/pull/913/commits/5f0ff27fbf70a12706bd3d915ea9360cbec3cc55